src/otio_aaf_adapter/adapters/advanced_authoring_format.py

```diff
@@ -1403,8 +1582,53 @@ def _simplify(thing):
         for c, child in enumerate(thing):
             thing[c] = _simplify(child)
 
+        if isinstance(thing, otio.schema.Track):
```

Comment:
Maybe we can break this into a separate function, we're getting pretty nested here.